### DISCUSSION TITLE
Personal Blogs/Lesson learnt based on activity

### DISCUSSION THREAD

**Discussion by vasnt** [assafelovic/gpt-researcher#739] [Ideas]
I am sure gpt-researcher is wonderful tool, though i havent yet tried it. 
I am wondering if it can perform below complex kind of task/chain-of-agents. 

- it should run in background
- observes my daily computer activity. wheaver i do  e.g. 
- if i ask few things to chatgpt/gemini/copilot
- if i write down some chats/emails
- if i perform usage of tool xyz using actions/tricks abc
- the tool(gpt-researcher) should monitor my activity & based on daily activity, it should generate a summarized blog/post/article with all the learning/things i performed to showcase/demonstrate my skills/ability on website/blog so others can read my progress.

**Comment by ElishaKay** [comment]
Hi @vasnt,

Gpt-researcher can analyze given websites or documents.

What you're describing is something like [the movie her with Joaquim Phoenix](https://youtu.be/dJTU48_yghs?si=RMHioS019euIQlHV) - great film, by the way.

Not quite there yet - maybe a Chrome Extension will eventually do that.

Best of luck

**Reply by vasnt** [reply]
- "given website/document" = cant it be configured (using some tool) so that it can fetch all docs i worked throughout the day?
- b2w, Crab Seems promising https://x.com/CamelAIOrg/status/1821970132606058943 or
- wordware
![image](https://github.com/user-attachments/assets/e2e676f8-6632-460a-b8bb-7eb599023a84)

**Reply by ElishaKay** [reply]
Sure.
If you find another tool to log your activity.
Just make sure you really trust that tool - since it might be a security breach of your sensitive info.

Or, think through what you want AI to know about you - maybe you'd like to summarize your research in a document and feed that into GPTR or provide some other limited context in a format GPTR takes as input (docx, text, csv...)

Set the DOC_PATH env variable to the relevant path with your activity logs.
